### ISSUE TITLE
the unweighted methylation frags is 8 in fig1 ? 

### ISSUE THREAD

**Issue by feihongloveworld** [JiejunShi/methylation_competition#1]
Why is the unweighted methylation frags equal to 8 in the fig1?

**Comment by JiejunShi** [comment]
A 'fragment' means consecutive dots in same color. In fig1, there are eight **_M_** fragments(solid dots), 5 in length 1, 2 in length 2, and 1 in length 4.

**Comment by feihongloveworld** [comment]
I got it, Thank you.   Weighted MCR was recommended in your paper, so I could get the MCR value in a block/region from the number of different state CpGs in the block/region?

another question, when I calculate an MCR value in a block, should I filter those reads which contain less than 4 CpGs in the block?

**Comment by JiejunShi** [comment]
> I got it, Thank you. Weighted MCR was recommended in your paper, so I could get the MCR value in a block/region from the number of different state CpGs in the block/region?

That's right. BTW, although some one call it 'MCR', we prefer to call it 'CAMDA'.

> another question, when I calculate an MCR value in a block, should I filter those reads which contain less than 4 CpGs in the block?

No, you don't need to. CAMDA can use all reads, which is one of its main advantages compared to window-based scores.
